app.py

```diff
@@ -100,178 +101,37 @@ def _project_list_query():
 import app_callbacks
 
 # CREATE APP OBJECT
-load_figure_template(["sandstone", "minty"])
+load_figure_template(["sandstone", "minty", "slate"])
+
+# stylesheet with the .dbc class
+dbc_css = "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
+
+plt_io.templates["custom_dark"] = plt_io.templates["slate"]
+plt_io.templates["custom_dark"]["layout"]["colorway"] = [
+    "#f8dd70",
+    "#c0bc5d",
+    "#8e9b4c",
+    "#62793d",
+    "#3c582d",
+    "#1c381d",
```

Comment:
an in-line comment here of which colors these are (like 'dark-green', 'light-green', 'sandy') would be nice so we easily update them w/o looking up the color codes.